recipes/paho-mqtt-c/all/conanfile.py

```diff
@@ -81,7 +83,7 @@ def package(self):
         self.copy("notice.html", src=self._source_subfolder, dst="licenses")
         cmake = self._configure_cmake()
         cmake.install()
-        tools.rmdir(os.path.join(self.package_folder, "lib", "cmake"))
+        tools.rmdir(os.path.join(self.package_folder, "lib", "cmake", "share"))
```

Comment:
This change caused the hook failure

What was your intention with this change?

Reply:
As a result of this exchange:

https://github.com/conan-io/conan-center-index/pull/2426#discussion_r466204939

Reply:
I see

`lib/cmake` has to be removed entirely, so the sub-directory `lib/cmake/share` gets removed already

The failure was about a root `share` directory, which installs documentation

```suggestion
        tools.rmdir(os.path.join(self.package_folder, "lib", "cmake"))
        tools.rmdir(os.path.join(self.package_folder, "share"))
```